data/human/news.txt

```diff
@@ -111,6 +113,19 @@ news "human merchant in human space"
 			"I was doing some jobs in the Core, when an enormous alien warship jumped in and attacked! People say I'm crazy, but I'm lucky to have gotten out of there with my life!"
 			"The Syndicate tries to keep alien attacks under wraps, but if you've been around as long as I have, you'll see them eventually."
 			"I was working a Syndicate defense job once, expecting to only deal with some pirates. Instead, I had to drive off an alien warship that jumped in same way the Quarg do! They made us sign a non-disclosure agreement, so don't tell anyone I said that."
+			"I once got hired by someone in search for his true love; he didn't know which ship she left in, so we travelled to dozens of planets to search for her, until he ran out of credits."
+			"I once had a passenger that returned to a spaceport where she met her true love. I wonder if they ever got together again."
+			"I once put 50,000 rifles and 150,000 grenades into my armory. One grenade got melted into the floor due to the weight of all the other weaponry on top of it and I could no longer sell that grenade."
+			"I felt lonely, so I participated in Romantic Matchmaking on a Syndicate planet, but that was a big scam! They never asked anything about the type of partner I was looking for, they only asked questions about what I wanted to buy in the next 10 years."
+			"I once joined Romantic Matchmaking organized by the Syndicate, but they couldn't tell me anything about my true love due to privacy concerns. Can you believe that?"
+			"Romantic Matchmaking as organized by the Syndicate is garbage; the Syndicate laws don't allow differentiating on gender, so there is a 50% chance that the match they find for you is the wrong gender."
+			"I once got married, but my partner didn't allow me to fly further than 5 systems away from her. Guess what? I'm now more than 5 systems away from my ex-partner."
+			"I once married somebody with no credits, and I lost half of my fleet when we divorced."
+			"I really found my true love in one of those Romantic Match Making programs; my love always joins me on every flight I do and we are a great team."
+			"I often join those Syndicate Romantic Matchmaking programs; they are a great source to pick-up crew members that work for free, at least until they find out. Then I dump them on whatever planet I'm visiting."
+			"Truck drivers in the old days were required by law to take a break every 2 hours. Can you imagine that in our business? Stopping our ships every 2 hours to take a rest?"
+			"I once had some Flat Earthers as passengers, they claimed that the planets we saw are 2 dimensional circles instead of 3 dimensional spheres."
```

Comment:
```suggestion
			"I once had some Flat Earthers as passengers. They claimed that the planets we saw are 2 dimensional circles instead of 3 dimensional spheres."
```